APIServer/appscale/api_server/log/messages.py

```diff
@@ -0,0 +1,292 @@
+""" LogService message containers. """
+
+from __future__ import division
+
+import base64
+import time
+from collections import deque
+
+import capnp
```

Comment:
This is the same thing we were doing before. I just tried to make our custom implementation more explicit. Here's our existing taskqueue stub: https://github.com/AppScale/appscale/blob/master/AppServer/google/appengine/api/logservice/logservice_stub.py#L21. I would like to remove capnp eventually.

Reply:
I think that capnp might be useful and that the main annoying thing about the logservice is not capnp but actually twistd.

Reply:
It might be useful if it were saving us any CPU when dealing with serializing and deserializing messages. At this time, the runtime serializes to protobuf, the API server deserializes protobuf and reserializes to capnp. Coming back out, it needs to deserialize from capnp and reserialize to protobuf. Since the runtime sends serialized protobuf, it would actually save us quite a bit of CPU to just keep that way.

On the Python side, it's an additional dependency that takes quite a long time to install. It also leaves us with extra message definitions and conversion code.